hedera-node/hapi/build.gradle.kts

```diff
@@ -25,7 +25,7 @@ configurations.all {
 }
 
 dependencies {
-    implementation("com.hedera.pbj:pbj-runtime:0.2.2")
+    implementation("com.hedera.pbj:pbj-runtime:0.3.0")
```

Comment:
We should define a property for the version (or create a lib)

Reply:
Agree, need to see if there is a way to do that. The version is in 2 places once for dependency here and once for plugin version. I wanted them both to be a single var but was not sure how to do it for plugin version. Need to try it.

Reply:
moved as much as possible to versionCatalogs